src/simulators/statevector/qubitvector.hpp

```diff
@@ -238,6 +238,10 @@ class QubitVector {
   // The matrix is input as vector of the column-major vectorized N-qubit matrix.
   void apply_matrix(const reg_t &qubits, const cvector_t &mat);
 
+  // Apply N-qubit matrixes to the state vector.
```

Comment:
Could you change this description to state it's applying fusion and only works for 1 and 2 qubit matrices.
Something like:
```
// Apply a N-qubit matrix constructed from composition of 1 and 2 qubit matrices.
// The sets of qubits and matrices are passed as vectors, where each individual matrix
// is input as a column-major vectorized matrix.
```